core/templates/dev/head/components/sharing_links_directive.html

```diff
@@ -1,40 +1,76 @@
 <script type="text/ng-template" id="components/sharingLinks">
   <ul class="oppia-sharing-links" layout="<[layoutType]>" layout-align="<[layoutAlignType]>">
     <li>
-      <a ng-href="https://plus.google.com/share?url=<[serverName]>/explore/<[explorationId]>" onclick="return !window.open(this.href, '', 'height=600, width=600, menubar=no, toolbar=no, resizable=yes, scrollbars=yes')" ng-click="registerShareExplorationEvent('gplus')" target="_window">
+      <a ng-href="https://plus.google.com/share?url=<[serverName]>/explore/<[explorationId]>" onclick="return !window.open(this.href, '', 'height=600, width=600, menubar=no, toolbar=no, resizable=yes, scrollbars=yes');" ng-click="registerShareExplorationEvent('gplus');" target="_window">
```

Comment:
You don't need to add the semicolons -- these lines were working fine as they were, I believe.